### DISCUSSION TITLE
GitHub docs don't have sufficient contrast

### DISCUSSION THREAD

**Discussion by melinath** [community/community#53989] [Accessibility] [answered]
### Select Topic Area

General

### Body

It's really hard to read the sidebars on the github docs due to the low contrast font color. Any way to increase the contrast, or is this an issue GitHub will need to resolve? This is using the latest version of Chrome.

From https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions:

![Screenshot 2023-04-28 at 10 39 15 AM](https://user-images.githubusercontent.com/299979/235216342-427df95a-7c1f-42f7-87a0-c3ebdde2f29c.png)
![Screenshot 2023-04-28 at 10 40 46 AM](https://user-images.githubusercontent.com/299979/235216625-5d64f1d6-b545-4d2a-bc08-0ec65cb50dd7.png)

**Comment by itstrangvu** [comment]
Upvoting this as I had to manually switch to dark mode in my Account. I suppose it has to do something with the dark mode tokens not being updated?

**Comment by adrianbienias** [comment]
Same issue here. Looks like docs are partially ignoring theme preferences set in account settings and displays theme based on autodetected system preferences.

**Comment by Emmarie-Ahtunan** [comment]
Have you looked at it recently? I just switched from dark mode to light mode and this is what I'm seeing on Chrome. I did, however, also turn off synch with system preferences. 

<img width="761" alt="Screenshot 2023-06-12 at 1 23 10 PM" src="https://github.com/community/community/assets/86572370/4316e502-c827-4a79-af7d-dc57c463e2ae">

**Comment by adrianbienias** [comment, marked as answer]
I checked and in my case the contrast looks ok now. Theme mode in [appearance settings](https://github.com/settings/appearance) set as sync with system or single theme, doesn't matter, in both cases GitHub and GitHub docs looks good in light/dark mode.

Looks like the issue has been fixed.

**Comment by cehfisher** [comment]
Hello @melinath - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/53989#discussioncomment-6155214). Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
